lib/BridgeAbstract.php

```diff
@@ -238,6 +245,52 @@ function($i){ return $i['name']; }, // Just display parameter names
 
 	}
 
+	/**
+	 * Loads configuration for the bridge
+	 *
+	 * Returns errors and aborts execution if the provided configuration is
+	 * invalid.
+	 *
+	 * @return void
+	 */
+	public function loadBridgeConfiguration() {
```

Comment:
Since this is a method of bridge, it can probably be shortened:

```suggestion
	public function loadConfiguration() {
```

Reply:
This method should also be mentioned in `BridgeInterface`.